src/main/resources/xlr_xldeploy/XLDeployClient.py

```diff
@@ -121,7 +123,7 @@ def get_parameter_names(self, parameter_type_id):
         parameter_names = []
         if params:
             for child in params:
-                parameter_names.append(child.get("name"))
+                parameter_names.append({'name': child.get("name"), 'default': child.get('default')})
```

Comment:
Can you also refactor `parameter_names` into `parameters`. And also rename the method into get_parameters?